### ISSUE TITLE
AnySoftKeyboard ghosting when exiting apk to home screen

### ISSUE THREAD

**Issue by m1chj** [duckduckgo/Android#1080]
**Describe the bug**
AnySoftKeyboard ghosting when leaving DDG with keyboard active at the moment of returning to home screen. This does not happen with Bromite or Iceraven. If I press back button, keyboard hides and then I go back to home screen, there's no ghosting.

**How to Reproduce**
1. Open DDG
2. Wait for keyboard to show up
3. Return to home screen
4. See keyboard ghosting

**Expected behavior**
See home screen right away without keyboard ghosting

**Smartphone):**
 - DDG App Version: 5.68.0 zs (56800)
 - Device: Huawei P40 Pro ELS-NX9 but this also happens on my Huawei P Smart 2019 POT-LX1
 - OS: Android 10 on both devices, EMUI 11.0.0 for P40 Pro and 10.0.0.229 for P Smart

Clip recorded on P40 Pro

https://user-images.githubusercontent.com/45080190/106254655-410ce280-6219-11eb-86dd-8644157fa992.mp4

**Comment by cmonfortep** [comment]
Thanks for the feedback and for the recording too, really helpful.

That was fixed in our 5.70.0 release. If you are using f-droid, please check this issue (https://github.com/duckduckgo/Android/issues/1069)

**Comment by m1chj** [comment]
@cmonfortep this still happens with newest version 5.76.0 downloaded from GitHub

https://user-images.githubusercontent.com/45080190/106464869-88090b00-6499-11eb-9572-48e9278f7e63.mp4

**Comment by cmonfortep** [comment]
Thanks. I can't reproduce it on my device.

I will try to reproduce it using the system keyboard. Just to double-check, are you using the system keyboard or another keyboard app installed in your phone?

**Comment by m1chj** [comment]
I'm using AnySoftKeyboard 1.10-r3 from F-Droid. It's also the only keyboard I have installed since I ADB'd whatever it was as a system default.

**Comment by m1chj** [comment]
@cmonfortep it's the same with OpenBoard keyboard

**Comment by cmonfortep** [comment]
Thanks, I managed to reproduce the issue on my device using that keyboard.

**Comment by Ammako** [comment]
I get this on 5.81.0 now, just with regular AOSP keyboard, except it stays up on the screen until you close it. It didn't happen before on older versions of the app.

**Comment by cmonfortep** [comment]
@Ammako we are trying to reproduce the scenario you mentioned, but the keyboard always goes away when sending the app to the background.

Would you mind giving us some more details (device, OS, current app version, ...)?
also if you could share the steps to reproduce it (or record a video/gif), it will really helpful. 

My steps are:
- open the app having an empty tab selected
- keyboard will automatically appear (URL bar focused)
- sending the app to the background.

but the keyboard always hides, am I missing something?

**Comment by Ammako** [comment]
I use a custom build of LineageOS 14.1 (7.1.2) on Nexus 5, it's not available online but I can upload it if needed. It comes with DuckDuckGo as a system app, and Bromite webview, in case that might be altering the behavior of the app (I wouldn't expect it to, though.)

Was on 5.81.0 but I'll let you know if I see it happen again on the current latest version. It only happens somewhat rarely, it doesn't happen just from switching the app away on an empty tab. I'll be browsing randomly and then I want to switch to another app to check an email or a message I just got, I use the recents app button to bring up my list of open apps and sometimes the keyboard goes away and then comes back and stays on, covering up most of the Recents screen.

**Comment by cmonfortep** [comment]
Sorry, wrong button when posting the comment :P

**Comment by Ammako** [comment]
Just had it happen again. 5.86.2, I started up the app and browsed to openweathermap, searched for my city, got the forecast for the day. Then I opened Recents view to quickly close the browser, and keyboard decided it felt like popping up and staying up on Recents screen, even though it wasn't even up on the page in the first place.

Might be a different issue from the one in this thread, though. Should I move this to a separate issue?
